dev/com.ibm.ws.jaxrs.2.1_fat_executor/test-applications/jaxrsapp/src/web/jaxrstest/JAXRSExecutorTestServlet.java

```diff
@@ -170,7 +171,10 @@ public void testCompletionStageRxInvokerSynchronousFunctionSwitchManagedExecutor
                 results.add(result);
             if (error != null)
                 results.add(error);
-            results.add(Thread.currentThread().getName());
+            String threadName = Thread.currentThread().getName();
+            results.add(threadName);
+            _log.info(m + " currentThread=" + threadName);
+            new Exception("handleAsync is running on " + threadName).printStackTrace(System.out);
```

Comment:
Was this code that dumps the stack meant to remain in the test code?   It's fine if it is.  I just wanted to draw your attention to it in case you meant to remove it before delivery.